tests/conftest.py

```diff
@@ -229,6 +229,12 @@ def empty_config(tmp_path) -> Config:
         configuration.CONFIG = current_config
         manager.load_profile(current_profile_name)
 
+        # This will set important global constants that are based on the location of the config folder. Without this,
+        # things like the :class:`aiida.engine.daemon.client.DaemonClient` will not function properly because the paths
+        # of the daemon files will be incorrect and still pointing to the path of the temporary config folder created by
+        # this fixture.
```

Comment:
```suggestion
        # This sets important global constants that are based on the location of the config folder. Without it,
        # things like the :class:`aiida.engine.daemon.client.DaemonClient` will not function properly after a test that uses this fixture
        # because the paths of the daemon files would still point to the path of the temporary config folder created by
        # this fixture.
```